app/helpers/provider_identity_permissions_helper.rb

```diff
@@ -12,6 +12,7 @@ module ProviderIdentityPermissionsHelper
   GROUP_PERMISSIONS = %w[create read].freeze
   INGEST_MANAGEMENT_ACL_PERMISSIONS = %w[read update].freeze
   NON_NASA_DRAFT_USER_PERMISSIONS = %w[create read update delete].freeze
+  NON_NASA_DRAFT_APPROVER_PERMISSIONS = %w[create read update delete].freeze
```

Comment:
Can we swap the orders of these to alphabetize?

Reply:
updated